lib/updatePlist.js

```diff
@@ -45,9 +44,40 @@ var getPlistConfigValue = function(plistConfig, manifest, key) {
   return value;
 };
 
+
+var replaceObjectRecursively = function (plistObj, manifest, plistConfig, key) {
+  var value;
+
+  Object.keys(plistConfig).forEach(function (nested_key) {
+    value = getPlistConfigValue(plistConfig, manifest, nested_key);
```

Comment:
```
var curr_val = plistConfig[nested_key];
if (curr_val is object) {
  replaceRecursive(plistObj, manifest, curr_val, key);
} else {
  value = getPlistConfigValue(plistConfig, manifest, nested_key);
  obj.setVal(plistObj, key.concat(nested_key).join("."), value);
}
```

wont this be enough?

Reply:
In that case Iterating won't be possible.